modules/gltf/src/lib/api/post-process-gltf.ts

```diff
@@ -135,6 +135,12 @@ class GLTFPostProcessor {
     this.images = images;
     this.jsonUnprocessed = json;
 
+    if (!options?.loadBuffers) {
+      // @ts-expect-error
+      const jsonPost: GLTFPostprocessed = {...json};
```

Comment:
It doesn't return postprocessed gltf that is anoying for a user. I don't agree with this change